api/tests/integration_tests/model_runtime/openai_api_compatible/test_llm.py

```diff
@@ -73,7 +73,9 @@ def test_invoke_stream_model():
         credentials={
             'api_key': os.environ.get('TOGETHER_API_KEY'),
             'endpoint_url': 'https://api.together.xyz/v1/',
-            'mode': 'chat'
+            'mode': 'chat',
+            "stream_mode_delimiter": "\\n\\n",
+            "stream_mode_exclude_chunks": "[DONE]"
```

Comment:
Can you add a test to check if it works without the stream_mode_delimiter and stream_mode_exclude_chunks?

Reply:
done  ,get the config with default

delimiter = credentials.get("stream_mode_delimiter", "\r\n")